<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-12-16T16:05:29 IST" name="Suite" finished-at="2021-12-16T16:07:21 IST" duration-ms="112324">
    <groups>
    </groups>
    <test started-at="2021-12-16T16:05:29 IST" name="Test" finished-at="2021-12-16T16:07:21 IST" duration-ms="112324">
      <class name="tests.LoginTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:tests.LoginTest@4ff8d125]" started-at="2021-12-16T16:05:29 IST" name="setup" finished-at="2021-12-16T16:06:07 IST" duration-ms="37803" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginWithValidCredentials()[pri:1, instance:tests.LoginTest@4ff8d125]" started-at="2021-12-16T16:06:07 IST" name="loginWithValidCredentials" finished-at="2021-12-16T16:06:09 IST" duration-ms="2714" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithValidCredentials -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LoginTest@4ff8d125]" started-at="2021-12-16T16:06:09 IST" name="tearDown" finished-at="2021-12-16T16:06:12 IST" duration-ms="2101" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:tests.LoginTest@4ff8d125]" started-at="2021-12-16T16:06:12 IST" name="setup" finished-at="2021-12-16T16:06:30 IST" duration-ms="18614" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginWithInvalidCredentials()[pri:2, instance:tests.LoginTest@4ff8d125]" started-at="2021-12-16T16:06:30 IST" name="loginWithInvalidCredentials" finished-at="2021-12-16T16:06:32 IST" duration-ms="2135" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at tests.LoginTest.loginWithInvalidCredentials(LoginTest.java:67)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithInvalidCredentials -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LoginTest@4ff8d125]" started-at="2021-12-16T16:06:33 IST" name="tearDown" finished-at="2021-12-16T16:06:34 IST" duration-ms="1601" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.LoginTest -->
      <class name="tests.SearchTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:tests.SearchTest@ca30bc1]" started-at="2021-12-16T16:06:34 IST" name="setup" finished-at="2021-12-16T16:07:03 IST" duration-ms="28302" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="searchWithAnExistingProduct()[pri:1, instance:tests.SearchTest@ca30bc1]" started-at="2021-12-16T16:07:03 IST" name="searchWithAnExistingProduct" finished-at="2021-12-16T16:07:04 IST" duration-ms="1192" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchWithAnExistingProduct -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.SearchTest@ca30bc1]" started-at="2021-12-16T16:07:04 IST" name="tearDown" finished-at="2021-12-16T16:07:06 IST" duration-ms="1866" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:tests.SearchTest@ca30bc1]" started-at="2021-12-16T16:07:06 IST" name="setup" finished-at="2021-12-16T16:07:18 IST" duration-ms="12210" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="searchWithNonExistingProduct()[pri:2, instance:tests.SearchTest@ca30bc1]" started-at="2021-12-16T16:07:18 IST" name="searchWithNonExistingProduct" finished-at="2021-12-16T16:07:19 IST" duration-ms="811" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchWithNonExistingProduct -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.SearchTest@ca30bc1]" started-at="2021-12-16T16:07:19 IST" name="tearDown" finished-at="2021-12-16T16:07:21 IST" duration-ms="2440" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.SearchTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
